app/controllers/observations_controller/index.rb

```diff
@@ -3,36 +3,10 @@
 # see observations_controller.rb
 module ObservationsController::Index
   # Displays matrix of all Observations, sorted by date.
-  # Searches come first because they may have the other params
-  # cop disabled per https://github.com/MushroomObserver/mushroom-observer/pull/1060#issuecomment-1179410808
-  # NOTE: ABC offense could be fixed with the technique described in
-  # https://github.com/MushroomObserver/mushroom-observer/pull/1060#issuecomment-1179469322
-  def index # rubocop:disable Metrics/AbcSize
-    # cop disabled because immediate fix is ugly and the offense is avoidable
-    # by fixing the ABC offense. See above
-    if params[:advanced_search].present? # rubocop:disable Style/GuardClause
-      advanced_search and return
-    elsif params[:pattern].present?
-      observation_search and return
-    elsif params[:look_alikes].present? && params[:name].present?
-      observations_of_look_alikes and return
-    elsif params[:related_taxa].present? && params[:name].present?
-      observations_of_related_taxa and return
-    elsif params[:name].present?
-      observations_of_name and return
-    elsif params[:user].present?
-      observations_by_user and return
-    elsif params[:location].present?
-      observations_at_location and return
-    elsif params[:where].present?
-      observations_at_where and return
-    elsif params[:project].present?
-      observations_for_project and return
-    elsif params[:by].present? || params[:q].present? || params[:id].present?
-      index_observation and return
-    else
-      list_observations and return
-    end
+  # NOTE: dipatch tables are in ObservationController
```

Comment:
The dispatch table must be a class instance variable of a subclass of ApplicationController.
But I defined it here, it's not a ivar of ObservationController.
@mo-nathan, @nimmolo: Is there a way to define those ivars here as ivars of ObservationsController?

Reply:
@mo-nathan Thanks for the review!
I will keep them in ObservationController.